doc/nrf/libraries/networking/nrf_cloud_agps.rst

```diff
@@ -33,47 +36,92 @@ Request and process A-GPS data
 
 A-GPS data can be requested using one of the following methods:
 
-* By specifying an array of A-GPS types
-* By requesting all the available assistance data
+* By specifying an array of A-GPS types.
+* By requesting all the available assistance data.
 
 If :kconfig:option:`CONFIG_NRF_CLOUD_MQTT` is enabled, the :c:func:`nrf_cloud_agps_request` function is used to request by type, and the :c:func:`nrf_cloud_agps_request_all` function is used to return all available assistance data.
 If :kconfig:option:`CONFIG_NRF_CLOUD_REST` is enabled, the :c:func:`nrf_cloud_rest_agps_data_get` function is used to request A-GPS data.
 
 When nRF Cloud responds with the requested A-GPS data, the :c:func:`nrf_cloud_agps_process` function processes the received data.
 The function parses the data and passes it on to the modem.
 
+Practical considerations
+************************
+
+When A-GPS data is downloaded using LTE network, the LTE link is in `RRC connected mode <RRC idle mode_>`_.
```

Comment:
I didn't change that text, only moved it. Yes, it looks like it would be wrong (I also had to check it myself), but the link is actually to this Wikipedia page: https://en.wikipedia.org/wiki/Radio_Resource_Control

I guess I could rename the link in `links.txt`, because the current name really is confusing.

Reply:
I've changed the link name.